CHANGELOG.md

```diff
@@ -4,12 +4,14 @@
 ### Added
 
 ### Changed
+- Minor UI Changes
 
 ### Deprecated
```

Comment:
should we remove the headers that have no bullet points?

Reply:
Good point, removed